<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="tree_widget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionScan_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScan_Directory"/>
   <addaction name="actionOpenTree"/>
   <addaction name="actionStopScanning"/>
   <addaction name="separator"/>
  </widget>
  <widget class="status_bar" name="statusBar"/>
  <action name="actionScan_Directory">
   <property name="icon">
    <iconset>
     <normaloff>dir/editor-trash-delete-recycle-bin-glyph-128.png</normaloff>dir/editor-trash-delete-recycle-bin-glyph-128.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scan Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>dir/Recycle_Bin_Empty.ico</normaloff>dir/Recycle_Bin_Empty.ico</iconset>
   </property>
   <property name="text">
    <string>Delete checked files</string>
   </property>
   <property name="toolTip">
    <string>Delete checked files?</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionStopScanning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop Scanning</string>
   </property>
   <property name="toolTip">
    <string>Stop Scanning</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>tree_widget</class>
   <extends>QTreeWidget</extends>
   <header>treewidget.h</header>
  </customwidget>
  <customwidget>
   <class>status_bar</class>
   <extends>QStatusBar</extends>
   <header>statusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
